### PR TITLE
docs(guides/best-practices/deployment-settings): first version 🚗  

### PR DESCRIPTION
**Pull request by qbzzt**
To be merged after https://github.com/latticexyz/mud/pull/2764 .

### CODE REVIEW

File: docs/pages/guides/best-practices/deployment-settings.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+## Before the deployment
+
+- Make sure you are using [the latest MUD version](https://mud.dev/changelog).
+- To make the `World`'s core contract [upgradeable](https://mud.dev/world/upgrades#making-an-upgradeable-world), set `deploy.upgradeableWorldImplementation` to `true` in [the `mud.config.ts` file](/config).
+- When writing the client use chain configs from [`@latticexyz/common/chains`](https://github.com/latticexyz/mud/tree/main/packages/common/src/chains): [Garnet (the testnet)](https://github.com/latticexyz/mud/blob/main/packages/common/src/chains/garnet.ts) and [Redstone (the production network)](https://github.com/latticexyz/mud/blob/main/packages/common/src/chains/redstone.ts).
+
+## During the deployment
+
+If possible, [deploy using AWS KMS](./kms).
+If not, [transfer ownership](https://github.com/latticexyz/mud/blob/main/packages/world/src/modules/init/implementations/AccessManagementSystem.sol#L50-L74) of the root namespace to AWS KMS as early as possible.
```
Comment (alvrs) [inline docs/pages/guides/best-practices/deployment-settings.mdx:16]:
```suggestion
If not, [transfer ownership](https://github.com/latticexyz/mud/blob/main/packages/world/src/modules/init/implementations/AccessManagementSystem.sol#L50-L74) of the root namespace to AWS KMS or a Multi-Sig wallet as early as possible.
```

File: docs/pages/guides/best-practices/deployment-settings.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## After the deployment
+
+- [Verify the contracts](/cli/verify) as soon as possible after deployment.
+  The more time passes, the more likely something is to change, which could make verification harder.
```
Comment (alvrs) [inline docs/pages/guides/best-practices/deployment-settings.mdx:21]:
```suggestion
```

**Review by alvrs — DISMISSED** [review]

**Comment by qbzzt** [comment]
@alvrs , can you reapprove, now I did what doesn't work in the Middle East (I resolved conflicts).

File: docs/pages/guides/best-practices/deployment-settings.mdx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+## Before the deployment
+
+- Make sure you are using [the latest MUD version](https://mud.dev/changelog).
+- To make the `World`'s core contract [upgradeable](https://mud.dev/world/upgrades#making-an-upgradeable-world), set `deploy.upgradeableWorldImplementation` to `true` in [the `mud.config.ts` file](/config).
+- When writing the client use chain configs from [`@latticexyz/common/chains`](https://github.com/latticexyz/mud/tree/main/packages/common/src/chains): [Garnet (the testnet)](https://github.com/latticexyz/mud/blob/main/packages/common/src/chains/garnet.ts) and [Redstone (the production network)](https://github.com/latticexyz/mud/blob/main/packages/common/src/chains/redstone.ts).
+
+## During the deployment
+
+If possible, [deploy using AWS KMS](./kms).
+If not, [transfer ownership](https://github.com/latticexyz/mud/blob/main/packages/world/src/modules/init/implementations/AccessManagementSystem.sol#L50-L74) of the root namespace to AWS KMS or a Multi-Sig wallet as early as possible.
```
Comment (holic) [inline docs/pages/guides/best-practices/deployment-settings.mdx:16]:
```suggestion
If not, [transfer ownership](https://github.com/latticexyz/mud/blob/main/packages/world/src/modules/init/implementations/AccessManagementSystem.sol#L50-L74) of the root namespace to AWS KMS or a multi-sig wallet as early as possible.
```

**Pull request merged**